### PR TITLE
this is square

### PR DESCRIPTION
**Pull request by arsalan366**
class that calculates area of a square

### CODE REVIEW

File: AreaCalc/AreaCalc/Square.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    {
+      
+        public void calculateSquareArea()
+        {
+            Console.Write("a = ");
+            int length = int.Parse(Console.ReadLine());
+            int area = length * length;
+            Console.Write("Square area = " + area);
+            
+
```
Comment (IceCoder1991) [inline AreaCalc/AreaCalc/Square.cs]: take out space

File: AreaCalc/AreaCalc/Square.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace Git_test
+{
+    class Square
+    {
+
```
Comment (IceCoder1991) [inline AreaCalc/AreaCalc/Square.cs:7]: SPace take out

File: AreaCalc/AreaCalc/Square.cs [hunk 3]
```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace Git_test
+{
+    class Square
+    {
+      
+        public void calculateSquareArea()
+        {
+            Console.Write("a = ");
```
Comment (IceCoder1991) [inline AreaCalc/AreaCalc/Square.cs]: What the point of this?

File: AreaCalc/AreaCalc/Square.cs [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+            int length = int.Parse(Console.ReadLine());
+            int area = length * length;
+            Console.Write("Square area = " + area);
```
Comment (KyleO249) [inline AreaCalc/AreaCalc/Square.cs]: Take out the console writing and just return the area as a decimal

**Pull request merged**